src/main/service/listManipulation/MagistratesStandardListService.ts

```diff
@@ -83,6 +39,56 @@ export class MagistratesStandardListService {
         return magsStandardListData;
     }
 
+    private processHearing(sitting, language) {
+        const allDefendants = [];
+        sitting['hearing'].forEach(hearing => {
+            hearing?.party?.forEach(party => {
```

Comment:
We are now inside the forEach loop for each hearing element, so the hearing is not optional anymore. Can remove the question mark after hearing.

Reply:
Question mark removed